src/pip/_vendor/vendor.txt

```diff
@@ -17,6 +17,6 @@ requests==2.25.1
 resolvelib==0.7.1
 setuptools==44.0.0
```

Comment:
It looks like [v45.0.0](https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v4500) dropped support for Python 2. Maybe that was the reason to stick with v44.0.0?

Reply:
Maybe, I don't know. In any case, pip only uses a small subset of setuptools that has not been updated for a while, and even that is going to be deprecated soon-ish, so let's not think too hard.

Reply:
Yes, it was Python 2. Yes, let’s not sweat about it.